README.md

```diff
@@ -42,11 +42,16 @@ import (
 )
 
 func main() {
-    // it requires a list of resolvers
-    resolvers := []string{"8.8.8.8:53", "8.8.4.4:53"}
+    // It requires a list of resolvers.
+    // Valid protocols are "udp", "tcp", "doh", "dot". Default are "udp".
```

Comment:
I think this should also be documented somewhere else, like making `parseResolver` an exported function.

Reply:
You are reviewing the README tho, @Mzack9999. :smiley:

Are you suggesting creating a separate section? I'd rather have this documented within the code so it can be served by godoc. Thoughts?

Reply:
lol yeah, I think it would be probably useful here, it's quite an important functionality
<img width="569" alt="Screenshot 2024-09-02 at 23 49 29" src="https://github.com/user-attachments/assets/a450e02f-a3d9-43e8-a31c-24ed77c15633">

Something like:
- supports resolvers ... (tcp, udp, doh, etc)